qa/full-cluster-restart/src/test/java/org/opensearch/upgrades/FullClusterRestartIT.java

```diff
@@ -1344,7 +1350,8 @@ public void testOperationBasedRecovery() throws Exception {
             if (minimumNodeVersion().before(Version.V_2_0_0)) {
                 settings.put(IndexSettings.INDEX_SOFT_DELETES_SETTING.getKey(), randomBoolean());
             }
-            createIndex(index, settings.build());
+            final String mappings = randomBoolean() ? "\"_source\": { \"enabled\": false}" : null;
```

Comment:
How is this change related to soft deletes?

Reply:
It's not related to this PR. While not detrimental this was cruft left over from me trying something unrelated. Thanks for catching. I've removed.